### PR TITLE
refactor: website

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What problem does this PR solve?

This PR refactor the website and give its a new interface.

### Does this PR introduce a user-facing change?

image

image

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
beautiful

**Comment by YangKeao** [comment]
Cool! The little spherical monster is very lovely 😸

File: website/README.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 ### Local Development
 
-```
-$ yarn start
+```sh
+yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
```
Comment (fewdan) [inline website/README.md:17]: It seems that "opens" is more reasonable?

Comment (g1eny0ung) [reply website/README.md:17]: Yep. It should be `opens`.

**Pull request merged**